### PR TITLE
New xUSDT theme created. 

### PR DESCRIPTION
**Pull request by YegorSan**
This PR is related to #2664 issue.

## Motivation

New layout for xUSDT theme was needed. 

### Enhancements
Created a new theme for xUSDT: 

screencapture-localhost-4000-2019-09-05-23_16_03

And dark mode:
screencapture-localhost-4000-2019-09-05-23_33_45

## Checklist for your PR

  - [ ] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so if necessary
  - [ ] If I added/changed/removed ENV var, I should update the list of env vars in https://github.com/poanetwork/blockscout/blob/master/docs/env-variables.md to reflect changes in the table here https://poanetwork.github.io/blockscout/#/env-variables?id=blockscout-env-variables. I've set `master` in the `Version` column.
  - [ ] If I add new indices into DB, I checked, that they don't redundant with PGHero or other tools

### CODE REVIEW

**Comment by YegorSan** [comment]
@vbaranov  Hello Victor. I want to clarify one question. As far as I can see , logic for rendering logo written in embedded elixir syntax: 

> "** alt="<%= subnetwork_title() %>" />

I've created new theme and uploaded logos, please confirm, that no additional steps must be performed from my end. Thank's in advance.

**Comment by vbaranov** [comment]
@YegorSan in addition to my review, CHANGELOG entry is missing

**Comment by YegorSan** [comment]
@vbaranov Hello. You are absolutely right. I've just created the new theme, but didn't add any additional changes to network selector yet, as it wasn't mentioned in this issue. Let me work on it right now. I'll take a further look and provide an update shortly.

**Comment by YegorSan** [comment]
@vbaranov  I've added new icons in styles for xUSDT, commented line to non-critical css, style for preload and style for icon. Please review. Thank you in advance.

File: apps/block_scout_web/assets/css/components/_network-selector.scss [hunk 1]
```diff
@@ -280,6 +280,9 @@ $network-selector-item-icon-dimensions: 30px !default;
   &-lukso-l14-testnet {
     background-image: url(/images/network-selector-icons/lukso-l14-testnet.svg)
   }
+  &-xusdt-mainnet {
```
Comment (vbaranov) [inline apps/block_scout_web/assets/css/components/_network-selector.scss]: @YegorSan please change the class name from `&-xusdt-mainnet` to `&-xusdt-chain` to conform it with xDai Chain

Comment (YegorSan) [reply apps/block_scout_web/assets/css/components/_network-selector.scss]: @vbaranov  Please give me a few minutes.

Comment (YegorSan) [reply apps/block_scout_web/assets/css/components/_network-selector.scss]: @vbaranov  Done. Please check.

**Pull request merged**